File: sentimental-mario-more/mario.py
from cs50 import get_int

# Get integer 1 to 8 inclusive
while True:
    num = get_int("Height: ")
    if num >= 1 and num <= 8:
        break

# Print #
for i in range(1, num + 1):
    print(" " * (num-i) + "#" * i + "  " + "#" * i)

File: mario/more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

    // Get height
    int h = get_int("Height: ");
    while (h > 8 || h < 1)
    {
        h = get_int("Height: ");
    }

    // Print mario
    for (int i = 0; i < h; i++)
    {
        // First space
        for (int j = 0; j < h - 1 - i; j++)
        {
            printf(" ");
        }

        // First #
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }

        // Second space
        printf("  ");

        // Second #
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }

        printf("\n");
    }
}
